app/templates/jinja2/dashboard/player.html

```diff
@@ -11,7 +11,9 @@
 {% block subcontent %}
 <div class="ui-grid s-dashboard__index">
   <div class="ui-grid">
-    {{ card("My Code", participant.code, "ui-card--display") }}
+    {% call card("My Code") %}
+    <a href="{{ url('player_code') }}">participant.code</a>
```

Comment:
![image](https://user-images.githubusercontent.com/17097380/90988332-044d0900-e560-11ea-9c46-7cc4f9f8b3a2.png)

Bruh.

Reply:
Oof my bad. There’s a relatively simply fix that I can push later today